### ISSUE TITLE
Bandplans

### ISSUE THREAD

**Issue by smurphboy** [smurphboy/OARCLogger#29]
FT8, FT4, APRS or Sat frequencies in an editable table. Anything that is like a band plan spot frequency.

Used when you set the mode in QSO to suggest a frequency in the field for you automatically. It can be overridden.

**Comment by MartinCRichards** [comment]
yes please, but remembering that people occasionally go off piste on a very crowded band

**Comment by smurphboy** [comment]
The plan is that its a default. So if you create an FT8 event then it defaults to the FT8 frequency for the band but you can overwrite.
